python/src/uagents/registration.py

```diff
@@ -32,11 +32,21 @@ async def register(
         pass
 
 
+class AgentGeoLocation(BaseModel):
+    # Latitude and longitude of the agent
+    lat: float
+    lon: float
```

Comment:
I propose to either write `long` or `lng` as these are the most common abbreviations or go for the full words to make it more specific.

Reply:
Actually lat/lon use is one of the most common ways to express. Anyways updated to full words.